From bbed941eb704b38912899eaa0eece70cc26529db Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:07:13 -0400
Subject: [tools] perf: Remove MAX_COUNTERS define from perf.h

Message-id: <1410181752-25631-189-git-send-email-jolsa@redhat.com>
Patchwork-id: 91903
O-Subject: [PATCH RHEL7.1 BZ1134356 188/307] perf tools: Remove MAX_COUNTERS define from perf.h
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 3dd422b4a5009e7654846534d535758d4cc985db
Author: Jiri Olsa <jolsa@kernel.org>
Date: Mon May 5 12:26:14 2014 +0200

description
===========
As it's no longer used.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/perf.h b/tools/perf/perf.h
index 5aa8ac4..3bea2e6 100644
--- a/tools/perf/perf.h
+++ b/tools/perf/perf.h
@@ -231,7 +231,6 @@ sys_perf_event_open(struct perf_event_attr *attr,
  return fd;
 }
 
-#define MAX_COUNTERS   256
 #define MAX_NR_CPUS   256
 
 struct ip_callchain {
-- 
1.7.1